openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-bucketexplorer/src/tools/BucketExplorer/BucketExplorer.vue

```diff
@@ -127,9 +133,17 @@
         </template>
         <template v-slot:item.action="{ item }">
           <v-icon
+            v-if="item.icon === 'mdi-file' && !isBinary(item.name)"
```

Comment:
Would be better to have isText instead of isBinary.  That way you know a file is viewable, rather than just excluding 4 extensions.